server/common/Params.h

```diff
@@ -89,6 +90,12 @@ typedef int IncidentTrackerIdType;
 typedef uint64_t ActionLogIdType;
 #define FMT_ACTION_LOG_ID PRIu64
 
+typedef uint64_t LastInfoIdType;
```

Comment:
I've fixed the corresponding column (last_info_id) type and length: https://github.com/project-hatohol/hatohol/commit/06101c718e2fad7a66c235f5e631916f2511d9ca.